example/ck_tile/17_grouped_gemm/grouped_gemm.hpp

```diff
@@ -182,7 +182,8 @@ struct GemmConfigPreshuffleDecode : public GemmConfigBase
     static constexpr auto Scheduler            = ck_tile::GemmPipelineScheduler::Default;
     static constexpr ck_tile::index_t Pipeline = CK_TILE_PIPELINE_PRESHUFFLE_V2;
     static constexpr bool Preshuffle           = true;
-    static constexpr bool DoubleSmemBuffer     = true;
+    static constexpr bool Persistent = false; // Preshuffle doesn't support persistent mode yet
```

Comment:
should this not be true now that you've added support for it?
```suggestion
    static constexpr bool Persistent = true;
```

Reply:
Ack